code/modules/research/xenobiology/crossbreeding/_misc.dm

```diff
@@ -0,0 +1,170 @@
+
+/obj/item/reagent_containers/food/snacks/rationpack/checkLiked(fraction, mob/M)	//Nobody likes rationpacks. Nobody.
+	if(last_check_time + 50 < world.time)
+		if(ishuman(M))
+			var/mob/living/carbon/human/H = M
+			if(H.mind && !H.has_trait(TRAIT_AGEUSIA))
+				to_chat(H,"<span class='notice'>That didn't taste very good...</span>") //No disgust, though. It's just not good tasting.
+				GET_COMPONENT_FROM(mood, /datum/component/mood, H)
+				if(mood)
+					mood.add_event(null,"gross_food", /datum/mood_event/gross_food)
+				last_check_time = world.time
+				return
+	..()
+
+//Ice stasis block - Chilling Dark Blue
+/obj/structure/ice_stasis
+	name = "ice block"
+	desc = "A massive block of ice. You can see something vaguely humanoid inside."
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "frozen"
+	density = TRUE
+	max_integrity = 100
+	armor = list("melee" = 30, "bullet" = 50, "laser" = -50, "energy" = -50, "bomb" = 0, "bio" = 100, "rad" = 100, "fire" = -80, "acid" = 30)
+
+/obj/structure/ice_stasis/Initialize()
+	. = ..()
+	playsound(src, 'sound/magic/ethereal_exit.ogg', 50, 1)
+
+/obj/structure/ice_stasis/Destroy()
+	for(var/atom/movable/M in contents)
+		M.forceMove(loc)
+	playsound(src, 'sound/effects/glassbr3.ogg', 50, 1)
+	return ..()
+
+//Gold capture device - Chilling Gold
+/obj/item/capturedevice
+	name = "gold capture device"
+	desc = "Bluespace technology packed into a roughly egg-shaped device, used to store nonhuman creatures. Can't catch them all, though - it only fits one."
+	w_class = WEIGHT_CLASS_SMALL
+	icon = 'icons/obj/slimecrossing.dmi'
+	icon_state = "capturedevice"
+
+/obj/item/capturedevice/attack(mob/living/M, mob/user)
+	if(contents.len)
+		to_chat(user, "<span class='warning'>The device already has something inside.</span>")
+	if(!isanimal(M))
+		to_chat(user, "<span class='warning'>The capture device only works on simple creatures.</span>")
+		return
+	if(M.mind)
+		to_chat(user, "<span class='notice'>You offer the device to [M].</span>")
+		if(alert(M, "Would you like to enter [user]'s capture device?", "Yes", "No") == "Yes")
+			if(user.canUseTopic(src, BE_CLOSE) && user.canUseTopic(M, BE_CLOSE))
+				to_chat(user, "<span class='notice'>You store [M] in the capture device.</span>")
+				to_chat(M, "<span class='notice'>The world warps around you, and you're suddenly in an endless void, with a window to the outside floating in front of you.</span>")
+				store(M, user)
+			else
+				to_chat(user, "<span class='warning'>You were too far away from [M].</span>")
+				to_chat(M, "<span class='warning'>You were too far away from [user].</span>")
+		else
+			to_chat(user, "<span class='warning'>[M] refused to enter the device.</span>")
+			return
+	else
+		if(istype(M, /mob/living/simple_animal/hostile) && !("neutral" in M.faction))
+			to_chat(user, "<span class='warning'>This creature is too aggressive to capture.</span>")
```

Comment:
No wiggle room here in simply converting particular aggressive groups like ``lavaland`` faction monsters to neutral? This would restrict you to being able to lazarus injector anything with a corpse and stuff into your poke'ball typically or by a liberal use of gold slimes/cult golem conversion.

- Not a critism just pointing out a area where it might have failed people's expectations if the first thing they were to do was to exit the lab and run around throwing it at native hostile faction mobs.

Reply:
If we want to go full pokemon, you could make it so no factioned/neutral mobs have a certain percent chance of being catchable that was inveresly proportional to their strength (attack*health or something), e.g. the stronger the month harder to catch. Than, upon catch, the mob is converted to your faction.

Reply:
Maybe add a function that lets you put a ghost into the thing you catch?

Reply:
I considered the above, but figured it might end up being a bit too powerful. Besides, there's several potions in place to remove hostile factions from creatures already in Crossbreeding.